roles/nginxplus/files/conf/http/cdh_prod_web.conf

```diff
@@ -80,6 +110,39 @@ server {
     }
 }
 
+server {
+    listen 443 ssl;
+    http2 on;
+    server_name cdh-beta.princeton.edu;
+
+    client_max_body_size 50M;
+
+    ssl_certificate            /etc/letsencrypt/live/cdh-web/fullchain.pem;
+    ssl_certificate_key        /etc/letsencrypt/live/cdh-web/privkey.pem;
+    ssl_session_cache          shared:SSL:1m;
+    ssl_prefer_server_ciphers  on;
+
+    location / {
+        proxy_pass http://beta_prod_cdhweb;
+        proxy_set_header Host $http_host;
+        proxy_set_header X-Forwarded-Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_cache prod_cdhwebcache;
+        proxy_connect_timeout      2h;
+        proxy_send_timeout         2h;
+        proxy_read_timeout         2h;
+        proxy_max_temp_file_size 0;
+        proxy_buffer_size          128k;
+        proxy_buffers              4 256k;
+        proxy_busy_buffers_size    256k;
+        proxy_ignore_client_abort on;
+        proxy_intercept_errors on;
+        health_check interval=10 fails=3 passes=2;
```

Comment:
is it missing the config to restrict to campus IPs / VPN or am I just not seeing it?